x-pack/solutions/observability/plugins/slo/ui_tests/services/slo_data_service.ts

```diff
@@ -5,29 +5,17 @@
  * 2.0.
  */
 
-import { KbnClient } from '@kbn/test';
 import { cli, DEFAULTS } from '@kbn/data-forge';
+import { KbnClient } from '@kbn/scout-oblt';
 
-export class SLoDataService {
-  kibanaUrl: string;
-  elasticsearchUrl: string;
-  params: Record<string, any>;
-  requester: KbnClient['requester'];
+export class SLODataService {
+  constructor(
+    private kibanaUrl: string,
+    private elasticsearchUrl: string,
+    private kbnClient: KbnClient
+  ) {}
```

Comment:
I assume it might be useful for other oblt plugins, could we add it as worker fixture similar to https://github.com/elastic/kibana/blob/7e0601a2e681664966e53fde84e1d56c8912e8aa/src/platform/packages/shared/kbn-scout/src/playwright/fixtures/worker/synthtrace.ts#L52-L95 in `kbn/scout-oblt`

Reply:
This way the  test code is cleaner and zero chance to miss the valid es host and other arguments